cssdata.py

```diff
@@ -0,0 +1,88 @@
+# import from existing libraries
+import pandas as pd
+from matplotlib import pyplot as plt
+
+# import from my modules
+from import_data import data_dirs
+
+
+def csv_to_dataframe(basedir, experiments, drs):
+    """
+    make a list to contain dataframes for the whole data
+    :param basedir: file directory where the raw data is saved
+    :param experiments: a list of integer that specifies the experiment numbers that you are targeting
+    :param drs: relative density of each trial
+    :return: a list of dataframes that contains each `.csv` file
+    """
+    dfs = []  # make an empty list to contain dataframes
+
+    # get the number of trials for each experiment
+    for experiment in experiments:
+        datadirs = data_dirs(experiment, basedir=basedir)  # get dir list of trial files in each exp
+        num_experiments = len(experiments)  # get num of experiments
+        num_trials = len(datadirs)  # get num of trials in each exp
+
+        # make a list to contain a dataframes for each trial in the exp
+        df_trial = []
+
+        # get dataframes for a single trial and append it to `df_trial`
+        for trial in range(num_trials):
+
+            # make a dataframes for each trial
+            df_single = pd.read_csv(datadirs[trial], header=5)  # make `.csv` to pandas `df`
+
+            # insert Dr values at the first column of the dataframes
+            df_single.insert(0, "Dr [%]", drs[experiment - 7][trial])
+
+            # compute and insert ru at the 4th column of the dataframes
+            # confining pressure of the test is the first (starting) effective vertical stress
+            confining_pressure = df_single.iloc[0, 4]  # get conf pressure of the trial
+            ru = df_single['Excess Pore Pressure [kPa]']/confining_pressure
+            df_single.insert(5, "ru", ru)
+
+            # Append this dataframes for the trial to `df_trial`
+            df_trial.append(df_single)
+
+        # Append `df_trial` to `dfs`
+        dfs.append(df_trial)
+
+    return dfs
+
+
+def plot_trial(dataframes, expindex, trialindex):

```

Comment:
If this function is supposed to plot the result of a single trial, shouldn't you just pass the trail object? Don't do figuring out where the trial data is inside this function. This function should get the trial data and plot - that's all. No "exp_index" or "trial_index"